src/modules/netinstall/NetInstallPage.cpp

```diff
@@ -62,4 +63,21 @@ void
 NetInstallPage::onActivate()
 {
     ui->groupswidget->setFocus();
+
+    // The NetInstallSelect global sotrage value can be used to make additional items selected by default
+    Calamares::GlobalStorage* gs = Calamares::JobQueue::instance()->globalStorage();
+    if ( gs->contains( "NetinstallSelect" ) && gs->value( "NetinstallSelect" ).canConvert( QVariant::StringList ) )
+    {
+        const QStringList selectNames = gs->value( "NetinstallSelect" ).toStringList();
+
+        static_cast< PackageModel* >( ui->groupswidget->model() )->setSelections( selectNames );
+    }
+
+    // If NetInstallAdd is found in global storage, add those items to the tree
+    if ( gs->contains( "NetinstallAdd" ) && gs->value( "NetinstallAdd" ).canConvert( QVariant::List ) )
+    {
+        const QVariantList groups = gs->value( "NetinstallAdd" ).toList();
+
+        static_cast< PackageModel* >( ui->groupswidget->model() )->appendModelData( groups, "packageChooser" );
```

Comment:
You might need `QStringLiteral()` there around the string, for people who compile with `QT_NO_CAST_ASCII` (I should add that to the test-suite)

Reply:
Ahh.....why do I have hardcoded reference to the packagchooser in the netinstall module?  I am pretty sure I meant to pass that data in.  I should definitely fix that.